timescaledb/tutorials/grafana/visualizations/candlestick.md

```diff
@@ -124,23 +125,23 @@ difference gives the traded volume for that bucket.
 ### Showing transaction volumes in a candlestick plot
 
 1.  Create a new candlestick panel with the following query:
-    ```sql
-    SELECT
-      time_bucket('$bucket_interval', time) AS time,
-      symbol,
-      FIRST(price, time) AS "open",
-      MAX(price) AS high,
-      MIN(price) AS low,
-      LAST(price, time) AS "close",
-      MAX(day_volume) - LAG(max(day_volume), 1) OVER(
-        PARTITION BY symbol
-        ORDER BY time_bucket('$bucket_interval', time)
-      ) AS bucket_volume
-    FROM stocks_real_time
-    WHERE symbol =  $symbol
-      AND time > $__timeFrom()::timestamptz and time < $__timeTo()::timestamptz
-    GROUP BY time_bucket('$bucket_interval', time), symbol;
-    ```
+      ```sql
```

Comment:
This looked great before actually! The overall indentation isn't necessary

Reply:
Ah I see, I got it wrong

Reply:
Just updated, please, can you confirm again?
Sorry for the stress 😊

Reply:
Sorry for the delay, this question got buried in my GH notifications! Looks great 😄 

No need to apologize! All is good ❤️ and this is really shaping up!